data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			`	"Glad you could make it," Alondo greets you as you exit your ship. "Welcome to the secure port. Our home away from home."`
+			
+			label landed
+			choice
+				`	"What's with the warship?"`
+					goto warship
+				`	"What's so secure about it?"`
+				`	"Looks like a nice place. Let's get to that briefing you mentioned."`
+					goto briefing
+			`	"Well it's closed to the public, and as you can see," Alondo says while pointing up the the warship above you, "the Hai are guarding it well. There's no 'human embassy' or 'counsulate' for us to make use of, so the Hai decided to put one of their old court buildings to use, allowing us to use it as our base of operations while we're navigating this crisis."`
+			choice
+				`	"This entire building used to be a courthouse?"`
+				`	"It doesn't feel like much of a crisis to me."`
+					goto crisis
+				`	"Why the heavy guard? Are they expecting violence?"`
+					goto warship
+			`	"It was. From the little I've seen, the Hai have a rather, shall we say, labyrinthine system of government. This building was only a small part of that. Whatever the case, we have a briefing to get to," Alondo says while motioning to the door inside.`
+				goto briefing
+			label crisis
+			`	"Well... I could explain more, but that's what this briefing is for," Alondo says while motioning to the door inside.`
+				goto briefing
+			label warship
+			`	"As I mentioned earlier, their government is on edge at the moment. I'm sure they aren't expecting us to be violent, but they also want to be prepared for anything. Whatever the case, we have a briefing to get to," Alondo says while motioning to the door inside.`
+			label briefing
+			`	Alondo leads you to a meeting room that feels large for the number of people present: it looks like a lecture hall that could easily fit a hundred people, but there are only six here. Aside from Alondo and yourself, there is a shorter man in what you recognize as a Republic diplomatic uniform messing with a holoprojector that doesn't appear to be working, a taller man in a suit sitting next to a woman in a formal dress in the front row, and an older looking Hai woman sitting alone at the end of a row of seats.`
+			`	"Glad you finally decided to show up, Alondo," says the man in the front.`
+			`	"And you brought a guest," says the Hai.`
+			`	"I did," Alondo responds. "This is Captain <first> <last>. Member of the Free Worlds Council during the war. Helped repel the Pug invasion. A capable pilot. Am I missing anything?"`
+			choice
+				`	"That sounds like everything."`
+				`	"Just that I'm good looking."`
+			`	"There we go!" says the man who was fiddling with the holoprojector as it whirs to life. "No time for introductions. We're late enough as is. Everyone, please sit down."`
+			`	The holoprojector displays a series of graphs. You're no expert in data analysis, but a highlighted "chance of total crisis" at the bottom ranging from 61% to 96% does not look promising.`
```

Comment:
Just wondering, how are they generating this percentage?

Reply:
It's made up, just like half of all statistics are.

Reply:
I mean how are the Hai getting this?

Reply:
This is from Xilin, not the Hai.

Reply:
I imagine that the Hai government has a department of very dedicated statisticians whose sole purpose is to calculate the exact numbers of the chance they'll be public knowledge in human space...?

... does it particularly matter?

Reply:
> It's made up, just like half of all statistics are.

Did you make that up?

Reply:
> This is from Xilin, not the Hai.

OK, I didn't see the context.

Reply:
> I imagine that the Hai government has a department of very dedicated statisticians whose sole purpose is to calculate the exact numbers of the chance they'll be public knowledge in human space...?

Sure!


> ... does it particularly matter?

Probably not.